workflows/tasks/WXS-variant-calling/README.md

```diff
@@ -1,5 +1,16 @@
 ## DNA-Seq WGS Sanger variant calling
```

Comment:
looks like i forget to change the title. it should be WXS somatic variant calling.